packages/build/src/tracing/main.ts

```diff
@@ -43,7 +43,7 @@ export const startTracing = function (options: TracingOptions, logger: (...args:
 
   // Sets the current trace ID and span ID based on the options received
   // this is used as a way to propagate trace context from Buildbot
-  trace.setSpanContext(context.active(), {
+  return trace.setSpanContext(context.active(), {
```

Comment:
This is the actual change that ensures the `traceId` and `spanId` are using when initialising the first span.